src/app/components/MediaPlayer/AudioLoader/index.jsx

```diff
@@ -26,10 +26,11 @@ const spin = keyframes`
   }
 `;
 
+// Using pixels here as SMP does not change size based on user's font size
 const Spinner = styled.svg`
-  width: 4rem;
-  height: 4rem;
-  margin-top: 5rem;
+  width: 64px;
```

Comment:
We usually use relative values, is this going to scale with page zoom/text resize? Same for the height and margin top?

Reply:
Unsure why SMP not changing size affect this?

Reply:
Page zoom works as expected

Discussed the text-resize on slack, but for benefit of others:

- SMP and the loader share the same vertical space on the page.  Because SMP doesn't scale based on font size, the available space doesn't scale
- If we make the loader get progressively bigger inside a container of static size, eventually it becomes bigger than the container
- This is additionally problematic because SMP seems to load in a white background before it broadcasts its "media initialised" event - so if the loader is bigger than the container, the middle part of it becomes invisible for a period during the load sequence: https://www.loom.com/share/896058521e4f45fa8941146aa496fed3

Can still go with this though, if we still think this is preferred to just having the loader not respond to text-resize